api/server/sdk/status/errormessage/errormessage.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2019 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package errormessage
```

Comment:
Yeah, that's why I thought also, so I settled with `erroressage` because they are functions to manage `gRPC.Statu` and not objects to manage `SdkErrorMessage`.

Reply:
Ah I see. Makes sense then